README.md

```diff
@@ -675,23 +729,27 @@ The response will return an array that has been parsed from the following JSON:
     }
 ]
 ```
+####Parameters
+
 **referrer**
-: The id of the referring user for this credit transaction. Returns null if no referrer is involved. Note this id is the user id in developer's own system that's previously passed to Branch's identify user API call.
+: The id of the referring user for this credit transaction. Returns null if no referrer is involved. Note this id is the user id in a developer's own system that's previously passed to Branch's identify user API call.
 
 **referree**
-: The id of the user who was referred for this credit transaction. Returns null if no referree is involved. Note this id is the user id in developer's own system that's previously passed to Branch's identify user API call.
+: The id of the user who was referred for this credit transaction. Returns null if no referree is involved. Note this id is the user id in a developer's own system that's previously passed to Branch's identify user API call.
 
 **type**
-: This is the type of credit transaction
+: This is the type of credit transaction.
+
+1. _0_ - A reward that was added automatically by the user completing an action or referral.
+1. _1_ - A reward that was added manually.
+2. _2_ - A redemption of credits that occurred through our API or SDKs.
+3. _3_ - This is a very unique case where we will subtract credits automatically when we detect fraud.
 
-1. _0_ - A reward that was added automatically by the user completing an action or referral
-1. _1_ - A reward that was added manually
-2. _2_ - A redemption of credits that occurred through our API or SDKs
-3. _3_ - This is a very unique case where we will subtract credits automatically when we detect fraud
+### Get Referral Code
```

Comment:
We are working on renaming all of these methods to Promo Code, could you please update all references accordingly?

Referral URLs are staying the same, just Referral Code changes.